src/components/Input.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classes from './Input.module.css';
+
+const Input = () => (
+  <div className={classes.formControl}>
+    <h2>ADD NEW BOOk</h2>
+    <form className={classes.form}>
+      <input type="text" placeholder="Book title" required />
+      <input type="text" placeholder="Author" required />
+      <select name="category" id="category" required>
+        <option value="" disabled selected>
+          Category
+        </option>
+        <option value="Action">Action</option>
+        <option value="Economy">Economy</option>
+        <option value="Fiction">Science Fiction</option>
+        <option value="Adventure">Adventure</option>
+      </select>
+      <button type="button" className={classes.addBtn}>
```

Comment:
- [OPTIONAL] As pointed out by the previous code reviewer, please it will be nice to always specify the correct button attribute, as different browsers may use different default types. This [article](https://www.w3schools.com/tags/att_button_type.asp) throws more light on that.

Reply:
Thanks for the suggestion 🙂